Editor/Localization/ja.po

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2023 DeNA Co., Ltd.
```

Comment:
Set my name in copyright.
I would appreciate it if you could write the URL of the Anjin repo as a reference.

Reply:
Ahhhh, I'm very sorry about it. I missed it my self review...

Reply:
Fixed at [5656cf4](https://github.com/nowsprinting/test-helper.monkey/pull/47/commits/5656cf4d82459027cfcb7b33359ae9a44e408cf8).

Reply:
Added a copyright notice and a permission notice at  [9537b41](https://github.com/nowsprinting/test-helper.monkey/pull/47/commits/9537b417b125bbbf9707d5b1175e09db3a052e54).